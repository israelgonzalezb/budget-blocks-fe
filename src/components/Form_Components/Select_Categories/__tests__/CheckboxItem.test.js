
// Auto-generated do not edit


/* eslint-disable import/no-extraneous-dependencies */
/* eslint-disable no-undef */
import React from 'react';
import renderer from 'react-test-renderer';
import CheckboxItem from '../CheckboxItem';


describe('CheckboxItem test', () => {
  it('CheckboxItem should match snapshot', () => {
    const component = renderer.create(<CheckboxItem
       />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
    // hey there...
  });
});
