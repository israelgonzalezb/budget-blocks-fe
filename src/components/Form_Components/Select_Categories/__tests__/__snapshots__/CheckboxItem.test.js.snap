// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CheckboxItem test CheckboxItem should match snapshot 1`] = `
<div
  className="form-check"
>
  <label>
    <input
      className="form-check-input"
      type="checkbox"
    />
  </label>
</div>
`;
